CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Backward Compatibility Breaks
 ### Added
+* If not expicitly set, use `retry_on_conflict` from Client configuration in Bulk updates [#2184](https://github.com/ruflin/Elastica/pull/2184)
```

Comment:
Should we put this under "fixed"? sounds more like a bug fix but also happy to keep it under Added.

Reply:
First I added to that section. Then I moved to Added. TBH, I would leave it up to you.

Reply:
Lets keep it, I can always clean it up later.